js/lwf-loader.js

```diff
@@ -63,7 +64,11 @@
           myCallback();
         }, d);
       };
+    })();
+  }
 
+  if (global.cancelAnimationFrame === undefined || /iP(ad|hone|od).*OS 6/.test(userAgent)) {
+    (function() {
       global.cancelAnimationFrame = function(timer) {
```

Comment:
Thank you! Is it better to fix this part as below, instead of separately checking requestAnimationFrame and cancelAnimationFrame?
```diff
+++ b/js/lwf-loader.js
@@ -50,7 +50,9 @@
     global.msCancelAnimationFrame;
 
   /** apply polyfills for iOS6 devices */
-  if (global.requestAnimationFrame === undefined || /iP(ad|hone|od).*OS 6/.test(userAgent)) {
+  if (global.requestAnimationFrame === undefined
+      || global.cancelAnimationFrame === undefined
+      || /iP(ad|hone|od).*OS 6/.test(userAgent)) {
     (function() {
       var vsync = 1000 / 60;
       var t0 = global.performance.now();
```